site/routes/definitions.js

```diff
@@ -64,10 +64,11 @@ export const routeDefinitions : Array<RouteDefinition> = [
     gen: genBadgersParams,
   },
   {
-    title: 'Badger',
+    title: ({ badger }) => badger.firstName + ' ' + badger.lastName,
```

Comment:
You should probably also do `[firstName, lastName].join(" ")` here like you did previously